db/migrate/20240722082922_add_shared_benefit_column_to_applicant.rb

```diff
@@ -0,0 +1,12 @@
+class AddSharedBenefitColumnToApplicant < ActiveRecord::Migration[7.1]
+  def up
+    add_column :applicants, :shared_benefit_with_partner, :boolean
+    Partner.where.not(shared_benefit_with_applicant: nil).each do |partner|
+      partner.legal_aid_application.applicant.update!(shared_benefit_with_partner: partner.shared_benefit_with_applicant)
+    end
+  end
+
+  def down
+    remove_column :applicants, :shared_benefit_with_partner
+  end
```

Comment:
Up/Down vs Change method.

Is there a preferred method for migration is there a difference, I thought the change method was the new approach?

Reply:
Change is fine if you are making a simple change, add/remove a column.
As I needed to migrate the data in one direction, I used a separate up/down split :+1: